web-server/src/comment/comment.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Post } from '../post/post.entity';
+import {
+    Column,
+    CreateDateColumn,
+    Entity,
+    PrimaryGeneratedColumn,
+    Tree,
+    ManyToOne,
+    TreeChildren,
+    TreeParent
+} from 'typeorm';
+
+@Entity()
+@Tree("closure-table")
```

Comment:
nice choice, that's exactly what I had in mind

Reply:
😄 👍